tests/map.c

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <errno.h>
+#include <sys/mman.h>
+#include <libxnvme.h>
+#include <xnvme_be_vfio.h>
```

Comment:
builds are failing for macos and apline linux because of its inclusion. I don't see any need for this header file inclusion.